src/game/sound_init.c

```diff
@@ -75,7 +75,8 @@ static u32 sMenuSoundsExtra[] = {
     SOUND_AIR_BLOW_FIRE,
     SOUND_ENV_ELEVATOR4,
 };
-static s8 sPaintingEjectSoundPlayed = FALSE;
+
+s8 sPaintingEjectSoundPlayed = FALSE;
```

Comment:
if this is now no longer static and used in other files, the name should be changed from s to g